OpenRA.Game/Network/Order.cs

```diff
@@ -38,7 +38,8 @@ enum OrderFields : short
 		ExtraData = 0x20,
 		TargetIsCell = 0x40,
 		Subject = 0x80,
-		Grouped = 0x100
+		Grouped = 0x100,
+		LocalizedString = 0x200,
```

Comment:
Is this actually needed? Why can't the LocalizedMessage order transfer its info using TargetString?